core/src/main/java/org/infinispan/configuration/parsing/Parser.java

```diff
@@ -508,19 +532,19 @@ private void parseJGroupsStack(XMLExtendedStreamReader reader, ConfigurationBuil
       holder.addJGroupsStack(stackConfigurator, extend);
    }
 
-   private void parseJGroupsRelay(XMLExtendedStreamReader reader, ConfigurationBuilderHolder holder, EmbeddedJGroupsChannelConfigurator stackConfigurator) throws XMLStreamException {
+   private void parseJGroupsRelay(ConfigurationReader reader, ConfigurationBuilderHolder holder, EmbeddedJGroupsChannelConfigurator stackConfigurator) {
```

Comment:
Currently the only way to configure multiple `remoteSites` without a parse error is to have repeated "remoteSite" elements, which in the yaml format should result in the first instance of the duplicate element being ignored.

```yaml
          name: xsite
          extends: image-tcp
          relay.RELAY2:
            site: blah
            max_site_masters: 2
            can_become_site_master: true
          remoteSites:
            defaultStack: tcp
            remoteSite:
              name: "nyc"
            remoteSite:
              name: "ldn"
```

Expected (assuming parent element of list entries is no longer required):

```yaml
          name: xsite
          extends: image-tcp
          relay.RELAY2:
            site: blah
            max_site_masters: 2
            can_become_site_master: true
          remoteSites:
            defaultStack: tcp
            remoteSites:
              - name: "nyc"
              - name: "ldn"
```